src/promnesia/sources/pocket.py

```diff
@@ -7,22 +7,43 @@
 def index() -> Results:
     from . import hpi
     from my.pocket import articles
+
     # TODO use docstring from my. module? E.g. describing which pocket format is expected
 
     for a in articles():
-        loc = Loc.make(title='pocket', href=a.pocket_link)
+        title = a.json.get('resolved_title', 'pocket')
+        loc = Loc.make(title=title, href=a.pocket_link)
+        # Add a reverse locator so that the Promnesia browser extension shows a
+        # link on the Pocket page back to the original URL.
+        loc_rev = Loc.make(title=title, href=a.url)
         hls = a.highlights
         if len(hls) == 0:
             yield Visit(
                 url=a.url,
                 dt=a.added,
-                context=None,
+                context=a.json.get('excerpt', None),
```

Comment:
nit: let's extract `excerpt =` variable to avoid duplication?

Reply:
Done